### ISSUE TITLE
overfitting

### ISSUE THREAD

**Issue by leileihl123** [kongds/LightXML#9]
Have you ever encountered overfitting？Some parameters may need to be modified? The dataset is a self-made Chinese dataset.

**Comment by kongds** [comment]
I haven't encountered overfitting, the reason of overfitting maybe removing dropout after evaluation in each training epoch.

**Comment by leileihl123** [comment]
thanks
